craigslist/__init__.py

```diff
@@ -237,25 +247,53 @@ def customize_result(self, result, html_row):
         """ Add custom/delete/alter fields to result. """
         pass  # Override in subclass to add category-specific fields.
 
-    def geotag_result(self, result):
+    def geotag_result(self, result, soup):
         """ Adds (lat, lng) to result. """
 
         self.logger.debug('Geotagging result ...')
 
-        if result['has_map']:
-            response = requests_get(result['url'], logger=self.logger)
-            self.logger.info('GET %s', response.url)
-            self.logger.info('Response code: %s', response.status_code)
-
-            if response.ok:
-                soup = BeautifulSoup(response.content, 'html.parser')
-                map = soup.find('div', {'id': 'map'})
-                if map:
-                    result['geotag'] = (float(map.attrs['data-latitude']),
-                                        float(map.attrs['data-longitude']))
+        map = soup.find('div', {'id': 'map'})
+        if map:
+            result['geotag'] = (float(map.attrs['data-latitude']),
+                                float(map.attrs['data-longitude']))
 
         return result
 
+    def include_details(self, result, soup):
+        """ Adds description, images to result """
+
+        self.logger.debug('Adding details to result...')
+
+        body = soup.find('section', {'id': 'postingbody'})
+        result['body'] = body.text.strip()
+
+        img_matcher = re.compile(
```

Comment:
I don't see any reason to avoid regular expression, other than this being the only use of it. 

I can switch it to what you suggested and get rid of the re import. I'll just need to double check that there aren't any images on the page other than the uploaded ones, or filter based on them starting with "images.craigslist"